solved_exercises/dani/02-tuples.py

```diff
@@ -27,10 +30,17 @@
 tuple2 = 'd','e'
```

Comment:
let's be consistent within the same file, surround all tuples with parentheses.

Reply:
ok